openmetadata-ui/src/main/resources/ui/src/pages/ServiceDetailsPage/ServiceDetailsPage.tsx

```diff
@@ -293,7 +293,8 @@ const ServiceDetailsPage: FunctionComponent = () => {
         getEntityTypeFromServiceCategory(serviceCategory),
         decodedServiceFQN,
         servicePermission,
-        serviceDetails?.deleted ?? false
+        serviceDetails?.deleted ?? false,
+        getEntityName(serviceDetails)
```

Comment:
Can we pass `serviceDetails` instead of just the `name`? This way, if additional information is needed on the collate side in the future, we won't have to make further changes.

Reply:
@ShaileshParmar11 updated!